.github/ISSUE_TEMPLATE/changeoption.yml

```
@@ -0,0 +1,32 @@
+---
+description: Change an existing target.
+labels: [change-target]
+name: Change target
+title: "<TARGET>: <DESCRIPTION>"
```

This might be a copy-paste mistake:

```suggestion
description: Change an existing option.
labels: [change-option]
name: Change option
title: "<SCOPE>: <DESCRIPTION>"
```